pulsar-common/src/main/java/org/apache/pulsar/common/util/FutureUtil.java

```diff
@@ -55,6 +57,36 @@ public static CompletableFuture<Object> waitForAny(Collection<? extends Completa
         return CompletableFuture.anyOf(futures.toArray(new CompletableFuture[0]));
     }
 
+    public static CompletableFuture<Optional<Object>> waitForAny(Collection<? extends CompletableFuture<?>> futures,
```

Comment:
Let's write some comments to describe this method clearly.

Reply:
fixed.